index.html

```diff
@@ -38,5 +38,144 @@ <h2 class="Welcome-h2">I’am a Software Developer</h2>
+        </ul>
+        <button class="grid-click">See Project</button>
+      </div>
+      <div class="grid-body">
+        <h2 class="grid-head">Profesional Art <br> Printing Data</h2>
+        <p class="grid-text">A daily selection of privately <br> 
+          personalized reads; no accounts or <br> 
+          sign-ups required. has been the <br> 
+          industry's standard</p>
+        <ul class="grid-buttons">
+          <li class="grid-buttons-text"><a href="#">html</a></li>
+          <li class="grid-buttons-text"><a href="#">bootstrap</a></li>
+          <li class="grid-buttons-text"><a href="#">Ruby</a></li>
+        </ul>
+        <button class="grid-click">See Project</button>
+      </div>
+      <div class="grid-body">
+        <h2 class="grid-head">Profesional Art <br> Printing Data</h2>
+        <p class="grid-text">A daily selection of privately <br> 
+          personalized reads; no accounts or <br> 
+          sign-ups required. has been the <br> 
+          industry's standard</p>
+        <ul class="grid-buttons">
+          <li class="grid-buttons-text"><a href="#">html</a></li>
+          <li class="grid-buttons-text"><a href="#">bootstrap</a></li>
+          <li class="grid-buttons-text"><a href="#">Ruby</a></li>
+        </ul>
+        <button class="grid-click">See Project</button>
+      </div>
+      <div class="grid-body">
+        <h2 class="grid-head">Profesional Art <br> Printing Data</h2>
+        <p class="grid-text">A daily selection of privately <br> 
+          personalized reads; no accounts or <br> 
+          sign-ups required. has been the <br> 
+          industry's standard</p>
+        <ul class="grid-buttons">
+          <li class="grid-buttons-text"><a href="#">html</a></li>
+          <li class="grid-buttons-text"><a href="#">bootstrap</a></li>
+          <li class="grid-buttons-text"><a href="#">Ruby</a></li>
+        </ul>
+        <button class="grid-click">See Project</button>
+      </div>
+    </div>
+    
+  </section>
+  <section class="about-area">
+    <div class="about-content">
+    <h1 class="about-head">About me</h1>
+    <p class="about-text">Hello I’m a software developer! I can help <br> 
+      you build a product , feature or website <br> 
+      Look through some of my work and <br> 
+      experience! If you like what you see and <br> 
+      have a project you need coded, don’t <br> 
+      hestiate to contact me.</p>
+      <button class="resume">Get My Resume</button>
+      <hr>
+    </div>
+    <div class="lang-used">
+      <div class="lang-d">
+      <span class="diamond my-shape"></span>
+      <h2 class="items-h">Language</h2>
+    </div>
+      <ul class="items">
+        <li class="items-link"><a href="#">Javascript</a></li>
+        <li class="items-link"><a href="#">Ruby</a></li>
+        <li class="items-link"><a href="#">Html</a></li>
+        <li class="items-link"><a href="#">Css</a></li>
+      </ul>
```

Comment:
- [ ] It would be better if the technology categories are in a `<ul>` and each of them in a `<li>`. For example;
```
<ul>
  <li>Langruage</li>
    <ul>
      <li>Javascript</li>
    </ul>
  <li>Framework</li>
  <li>Skills</li>
</ul>

```
In this way!!! :muscle: